packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -134,43 +166,41 @@ export const getMaxMin = (
           let max = BigNumber.minimum(
             method.limits.max,
             isSddFlow ? sddLimit.max : pair.buyMax,
-            isSddFlow
-              ? sddLimit.max
-              : convertBaseToStandard('FIAT', limitMaxAmount, false)
+            isSddFlow ? sddLimit.max : convertBaseToStandard('FIAT', limitMaxAmount, false)
           ).toString()
 
-          if (
-            method.type === 'FUNDS' &&
-            sbBalances &&
-            limits?.maxPossibleOrder
-          ) {
+          let fundsChangedMax = false
+          if (method.type === 'FUNDS' && sbBalances && limits?.maxPossibleOrder) {
             const { available } = sbBalances[method.currency]
+            // available is always in minor string
+            const availableStandard = available
+              ? convertBaseToStandard('FIAT', available)
+              : available
```

Comment:
main change is on this line, we have to convert this to amount value and then to use it in compare logic